lib/gems/pending/appliance_console/cli.rb

```diff
@@ -322,6 +329,23 @@ def extauth_opts
       extauthopts.update_configuration(extauthopts_hash)
     end
 
+    def set_server_state
+      service = LinuxAdmin::Service.new("evmserverd")
+      service_running = service.running?
+      case options[:server]
+      when "status"
+        service_running ? "running" : "not running"
```

Comment:
Yeah...This before says the sate of running but we removed all the says methods so...If you want I can remove it  @carbonin